### DISCUSSION TITLE
Are there models trained in 'x0' mode?

### DISCUSSION THREAD

**Discussion by gaopinghai** [huggingface/diffusers#8776] [Q&A]
Hi, I need SD models trained in 'x0' mode. However, I can not find any pre-trained models. Please help, thanks!

**Comment by asomoza** [comment]
Probably there isn't that much if any models trained with "x0", you can refer to this [answer](https://github.com/CompVis/stable-diffusion/issues/489#issuecomment-1499204653) from the original repository as to why.

Also, this is a library for using those models, not for finding them. You’ll probably have better luck on sites or repositories where the community consists of people who train models.
